test-cases/manager/manager-regression-multiDC-set-distro.yaml

```diff
@@ -22,3 +22,5 @@ aws_instance_profile_name_db: 'qa-scylla-manager-backup-instance-profile'
 
 ami_monitor_user: 'centos'
 ami_id_monitor: 'ami-02eac2c0129f6376b' # Official CentOS Linux 7 x86_64 HVM EBS ENA 1901_01
+
+scylla_encryption_options: "{'key_provider': 'none'}"
```

Comment:
I think we should find the root cause before disabling KMS, this would disable KMS completely, and it might not be the desired outcome, depending on the scylla under test.

Reply:
Yep, true, encryption disabling is not the best way.
The issue is addressed in manager ticket https://github.com/scylladb/scylla-manager/issues/3829.
I'll remove this line of code.

Reply:
Removed encryption disabling.
@fruch @ShlomiBalalis Could you please review it once again?